<launch>
    <node name="pose_controller" pkg="dynamic_controller" type="pose_controller" output="screen" />
    <include file="$(find potential_field_planner)/launch/joint_planner.launch"/>
    <include file="$(find cubic_polynomial_planner)/launch/pose_planner.launch"/>

    <!--Stiffness and Damping-->
    <param name="/K_E" value="0.1"/>
    <param name="/D_E" value="0.1"/>
    <param name="/K_J" value="50"/>
    <param name="/D_J" value="5"/>

    <!--Subscribers-->
    <param name="/joint_states_topic" value="/gen3/joint_states"/>
    <param name="/reference_position_topic" value="/gen3/reference/position"/>
    <param name="/reference_velocity_topic" value="/gen3/reference/velocity"/>
    <param name="/reference_pose_topic" value="/gen3/reference/pose"/>
    <param name="/reference_twist_topic" value="/gen3/reference/twist"/>

    <!--Publisher-->
    <param name="/joint_group_effort_controller_command" value="/gen3/joint_group_effort_controller/command"/>
    <param name="/feedback_pose_topic" value="/gen3/feedback/pose"/>
    <param name="/feedback_twist_topic" value="/gen3/feedback/twist"/>

</launch>